scenes/world_map/scene_switcher.gd

```diff
@@ -0,0 +1,6 @@
+extends Node
```

Comment:
My only other nitpick would be that I don't think this should live in the `world_map` directory. I would expect this to be used to switch between the scenes that make up a quest, too, even though the `spawn_point` functionality would probably not be used, just so that we can have consistent transitions between scenes and (as in #50) do fancy stuff like putting the current scene into the URL hash in web builds.

In that branch I placed it at `res://scenes/scene_switcher/scene_switcher.gd` – there's no actual *scene* there but it seemed as good a home as any and it matches the `game_state.gd` script being in a `res://scenes/game_state` directory.